packages/mdx/mdx-hast-to-jsx.js

```diff
@@ -7,7 +7,8 @@ function toJSX(node, parentNode = {}, options = {}) {
   const {
     // Default options
     skipExport = false,
-    preserveNewlines = false
+    preserveNewlines = false,
+    wrapExport
```

Comment:
The one thing that worries me about the option is that there are now two options (`skipExport` and `wrapExport`) that determine what gets returned. Should these be completely separate or is there a situation where you might wrap the component but not export it?
```js
// should this be possible? and what would the wrapped variable be named?
const ??? = React.memo(MDXContent)
```
This could be written to support that, but I'm not sure whether or not it should be supported.

Reply:
Hypothetically these options should be mutually exclusive, so I'm not _super_ worried about the overlap. `skipExport` is used pretty rarely and only for advanced use cases.